lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -237,7 +237,7 @@ llvm::Error Caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
   }
   const std::string &opName = loadOperatorName(op);
 
-  if (typeName == "Conv" || typeName == "Int8Conv" ||
+  if (typeName == "Conv" || typeName == "ConvRelu" || typeName == "Int8Conv" ||
```

Comment:
@zrphercule Is this correct? Where are you constructing the Relu in the code below?

Reply:
Yeah, I think we need to create a ReLU for the float version. It only makes sense to skip the ReLU for the quantized version assuming that the Conv has non-negative quantization range.